test/Microsoft.ML.CpuMath.PerformanceTests/SsePerformanceTests.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using Microsoft.ML.Runtime.Internal.CpuMath;
+
+namespace Microsoft.ML.CpuMath.PerformanceTests
+{
+    public class SsePerformanceTests
+    {
+        private const int EXP_MAX = 127;
+        private const int EXP_MIN = 0;
+
+        private const int IDXLEN = 1000003;
+        private const int LEN = 1000003;
+        private const int EXP_RANGE = EXP_MAX / 2;
+        private const int DEFAULT_SEED = 253421;
+        private const float DEFAULT_SCALE = 1.11f;
+
+        private float[] src, dst, original, src1, src2;
```

Comment:
Nit: This all non constant privates should be prefixed with `_` but it can be updated in your next PR to not block this one.

Reply:
Updated the issue page (https://github.com/briancylui/machinelearning/issues/2) to include this action item for future follow-up.  Thank you for your input!